### ISSUE TITLE
pic32-gdb for OpenOCD/PlatformIO debugging support

### ISSUE THREAD

**Issue by lstandage** [chipKIT32/chipKIT-cxx#14]
In order to support debugging using PlatformIO, a version of GDB is needed in the tools. 

It is possible to use the version of GDB provided by Imagination, but it would be good to not require the user to have to get multiple tool sets.

**Comment by geoffwozniak-mchp** [comment]
What is the naming convention you require for the release images?

The current ones (without GDB) are available here: https://github.com/chipKIT32/chipKIT-compiler-builds/releases. Does anything need to be different?

**Comment by EmbeddedMan** [comment]
No, I don't think so. It would be best if it's version number is larger
than 1.43, but other than that it's up to you. Once the release images are
built and named, I'll point the chipKIT-core files to use the new compiler,
no matter what the filenames are.

*Brian
